File: src/main/php/Easy/HeightChecker.php
<?php

declare(strict_types=1);

namespace Src\Easy;

class HeightChecker
{
    public function transformHeightChecker(array $heights): int
    {
        $count = 0;
        $arr = [...$heights];
        sort($arr);
        foreach ($heights as $i => $value) {
            if ($arr[$i] !== $heights[$i]) {
                $count++;
            }
        }

        return $count;
    }

    public function solve(array $heights): int
    {
        return $this->transformHeightChecker($heights);
    }
}

File: lib/easy/height_checker.dart
int heightChecker(List<int> heights) {
  int count = 0;
  List<int> arr = List.from(heights);
  arr.sort();
  for (int i = 0; i < heights.length; i++) {
    if (heights[i] != arr[i]) {
      count++;
    }
  }
  return count;
}
